ui/components/app/assets/nfts/nft-details/nft-full-image.tsx

```diff
@@ -81,7 +82,7 @@ export default function NftFullImage() {
               <NftItem
                 src={isImageHosted ? image : nftImageURL}
                 alt={nftImageAlt}
-                name={name}
+                name={name ?? undefined}
```

Comment:
Is this change related to the multiple image fix? Just making sure it's intentional

Reply:
AH good catch, I was trying to sync all the types... but it was a bit overwhelming to do in 1 PR.
I think this sneaked through 😄

Reply:
Fixed https://github.com/MetaMask/metamask-extension/pull/30494/commits/5dabaf6f9c13b8b63d7453ddc95a01b0407b21e2